CMakeLists.txt

```diff
@@ -44,14 +44,11 @@ endif()
 
 if (CMAKE_COMPILER_IS_GNUCXX OR CMAKE_COMPILER_IS_CLANGCXX)
     set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wextra -Wall -Wstrict-aliasing=2")
```

Comment:
We may want to set the warning flags over all the local targets of Aikido. So `add_compile_options()` would be enough I think.